src/views/ManageList.jsx

```diff
@@ -1,7 +1,95 @@
+import { useState } from 'react';
+import { addItem } from '../api';
+
 export function ManageList() {
+	const [formData, setFormData] = useState({
+		name: '',
+		frequency: '',
+	});
+
+	// const [formSubmitted, setFormSubmitted] = useState(false);
```

Comment:
```suggestion
```
<!-- Nitpicks are trivial preference-based requests.  -->
**nitpick ( non-blocking ):** looks like you don't need this anymore?